### ISSUE TITLE
Want to Contribute

### ISSUE THREAD

**Issue by anish2210** [ArohiSoftwares/tiffin-management-system#18]
I would like to add some value to your repo through my contribution.... I saw @Siddheshuncodes post in the opportunity section 
![image](https://github.com/user-attachments/assets/24d080cb-7136-47d3-8f0c-0b172832df56)

@Siddheshuncodes  can you please let me know some changes that I can make on the application.

**Comment by Siddheshuncodes** [comment]
yes you can start Contributing just fork it and set up locally some code is not working so you will have to fix it and then get the initial app working then  I will assign you the further issues

**Comment by anish2210** [comment]
> yes you can start Contributing just fork it and set up locally some code is not working so you will have to fix it and then get the initial app working then I will assign you the further issues

I have setup the app locally now could you please let me know which part of the code is not working properly (although there are too many parts that are not working properly, but which part are you talking about?).
